clients/admin-ui/src/features/config-wizard/ConfigWizardWalkthrough.tsx

```diff
@@ -50,27 +47,23 @@ const ConfigWizardWalkthrough = () => {
 
   return (
     <>
-      <Box bg="white">
-        <Button
-          bg="transparent"
-          fontWeight="500"
-          m={2}
-          ml={6}
-          onClick={handleCancelSetup}
-        >
-          <CloseSolidIcon /> Cancel setup
-        </Button>
-      </Box>
+      {!features.navV2 && (
+        <Box bg="white">
+          <Button
+            bg="transparent"
+            fontWeight="500"
+            m={2}
+            ml={6}
+            onClick={handleCancelSetup}
+          >
+            <CloseSolidIcon /> Cancel setup
+          </Button>
+        </Box>
+      )}
       <Divider orientation="horizontal" />
```

Comment:
I think this divider also needs to be excluded in navV2:
https://www.loom.com/share/70e5da4e581b47c69b212739f626a6fc

Reply:
Issue has been resolved. Pull the latest. Thanks